proto/cosmos/feegrant/v1beta1/tx.proto

```diff
@@ -55,3 +58,14 @@ message MsgRevokeAllowance {
 
 // MsgRevokeAllowanceResponse defines the Msg/RevokeAllowanceResponse response type.
 message MsgRevokeAllowanceResponse {}
+
+// MsgPruneAllowances prunes expired fee allowances.
```

Comment:
Since feegrant v0.1

Reply:
That was going to be my next question, you are always one step ahead ⚡